pyocd/target/pack/cmsis_pack.py

```diff
@@ -1,6 +1,7 @@
 # pyOCD debugger
 # Copyright (c) 2019 Arm Limited
 # Copyright (c) 2020 Men Shiyun
+# Copyright (c) 2020 Fraunhofer-Gesellschaft zur Förderung der angewandten Forschung e.V.
```

Comment:
Unfortunately Python 2.7 doesn't like the non-ASCII `ö` character.

If you add `# -*- coding: utf-8 -*-` as the first line it will be fixed.

Reply:
Well ... Python2 :smile:

Reply:
Yeah… I'll be a happy person when it's time to completely drop Python 2 from pyOCD!